### DISCUSSION TITLE
How to control the Description of Packages (published via Gradle/maven)

### DISCUSSION THREAD

**Discussion by p-bakker** [community/community#10694] [Actions]
Hi,

We're publishing some Java libraries to GitHub Packages via gradle/maven, see https://github.com/mozilla/rhino/packages/1179324

I'm wondering how we are to control teh Description that is to be displayed un the package page under "About this package"

I'm sure it should somehow be possible to source that content from somewhere, instead of having to add that manually everytime after a package is published.

However I've not been able to find any pointers on how to do so.

Any thoughts appreciated!

**Comment by abbylentz** [comment]
Hi,

The description field in the POM file is used to populate "About this package". There is a bug, however, where the field is emptied for packages containing more than one file. Engineering is aware of the bug and has it scheduled to be resolved. Unfortunately, I'm not able to commit to a release window.

**Reply by p-bakker** [reply]
Tnx for the update

**Comment by wyvx** [comment]
Any updates on this issue?
It's been more than a year since the last official reply and the "About this package" section is still empty.

**Comment by endorh** [comment]
In my case, the package description is automatically generated, but it ignores my POM and instead uses the description from one of my dependencies (ANTLR 4)! That's even worse, because it actually forces me to manually correct the description to prevent confusion.
The description change then takes its sweet time to be reflected, and reverts whenever I upload a new version, which is just great :)

Since I use Gradle to generate my POM and publish the package, I just double checked that the POM that gets uploaded to GitHub has the correct information, and it does.
I have a limited understanding of Maven since I rarely interact with it directly, so I have no idea how this can even happen.

Add this issue to #26634, which has been ignored for ~3 years already, and my willingness to keep using GitHub packages starts to fade...

**Comment by Nikita-Guchakov** [comment]
Is there any workaround for this? I tried to add pom.xml with description at different places of my jar as WA, but had no luck, it either takes random dependency description or leaves it empty
